packages/@core.keystone/test.utils.js

```diff
@@ -368,6 +368,11 @@ const isMongo = () => {
     return conf.DATABASE_URL.startsWith('mongo')
 }
 
+const getRandomBetween = (min, max) => min + Math.floor(Math.random() * (max - min))
```

Comment:
There is lodash for that
https://lodash.com/docs/4.17.15#random

Reply:
fixed